ApiTests/Utils/PetStoreApiUtils.cs

```diff
@@ -4,33 +4,33 @@
 
 namespace TestProject.Utils
 {
-    internal class PetStoreApiUtils
+    static class PetStoreApiUtils
     {
-        private static readonly string GetEndpoint = "pet/";
-        private static readonly string PostEndpoint = "pet/";
+        private static readonly string PetEndpoint = "pet/";
 
         public static Pet GetPetById(long id)
         {
             return DeserializePetResponse(
-                ApiUtils.SendGetRequest(GetEndpoint + id));
+                ApiUtils.SendGetRequest(PetEndpoint + id));
         }
 
         public static RestResponse PostPet(Pet pet)
         {
-            return ApiUtils.SendPostRequest(PostEndpoint, pet);
+            return ApiUtils.SendPostRequest(PetEndpoint, pet);
         }
-        
-        public static bool PostPetIsSuccessful(Pet pet)
+
+        public static RestResponse PutPet(Pet pet)
         {
-            //implement a logic of sending a post request with a verification of success
-            return false;
+            return ApiUtils.SendPostRequest(PetEndpoint, pet);
         }
 
-        public static void DeletePetById(string id)
+        public static bool DeletePetById(string id)
         {
-            //add delete request for PetStore
+            RestResponse response = ApiUtils.SendDeleteRequest(PetEndpoint + id);
+            return response.StatusCode == System.Net.HttpStatusCode.OK;
```

Comment:
this method shouldn't have this validation in it, otherwise it's not reusable. 
For example, if we'd have a test case to check that 'deleting non-existing pet returns 404' we won't be able to reuse it in fill until we change it